### DISCUSSION TITLE
Dynamically registering field arrays?

### DISCUSSION THREAD

**Discussion by Heilemann** [react-hook-form/react-hook-form#8199] [Q&A] [answered]
I have a need to generate 0...n field arrays, each unique and defined by users. But since I can't fire the hook an arbitrary number of times, I'm not sure how I might do this (field array in a field array won't work for me).

**Comment by Heilemann** [comment, marked as answer]
Actually, I don't know why I didn't think about it, but I can just throw it into a component. Ugh. Braining is hard sometimes 😄

**Reply by Moshyfawn** [reply]
And writing your issue down can really help to come with the solution! \m/

But, yeah, braining **is** hard! I fully subscribe to this! xD

**Reply by bluebill1049** [reply]
Awesome that you sort it out 👍
